RocketApp/Domain/Tests/RocketsClientTests/RocketConverter+liveTests.swift

```diff
@@ -0,0 +1,80 @@
+import Dependencies
```

Comment:
Tenhle soubor se jmenuje `...+liveTests` a pak mas `ˇ...liveValueIntegrationTests`, to neni konzistentni, sjednot to, asi na `...+live...Tests`.

Reply:
A bacha, tohle je spis `RocketClientConvertersTests`.

Reply:
> A bacha, tohle je spis `RocketClientConvertersTests`.

Kdyz uz tak RocketDetailConverter mi dava smysl.

Reply:
Prejmenovano.

Reply:
Ta adresarova struktura v `Tests` musi 1:1 kopirovat strukturu `Sources`. Pokud je converter v Models > Converters, tak stejne to musi bejt i tady. Narovnej to prosimte i pro ostatni soubory.